app/sftp.py

```diff
@@ -0,0 +1,13 @@
+import pysftp
+import os
+
+
+def pull_videos(ip, username, password):
+    os.chdir('/home/pi/Videos')  # Set pi 3 cwd to Videos
+    remoteCam = pysftp.Connection(ip, username=username, password=password)
+    remoteCam.chdir('/home/pi/Videos')  # Set pi 0 cwd to Videos
+    files = remoteCam.listdir()
+    for file in files:
+        if '.mp4' in file and not '1stPASS' in file:
```

Comment:
What is the meaning of `1stPASS`? Maybe add a comment in the code.

Reply:
1stPASS is a intermediate file made during video capture